### DISCUSSION TITLE
Are data-* attributes supported?

### DISCUSSION THREAD

**Discussion by shawngustaw** [facebook/stylex#130] [Q&A] [answered]
Currently evaluating stylex + radix ui and they use html data attributes like `data-state="open"`  (i.e. https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes). Is there a way to support this in stylex?

**Comment by nmn** [comment, marked as answer]
On the element itself, this can be done with `:is([data-state='open'])` where you would otherwise use `:hover`. This is not a recommended pattern, however since it generates new unique CSS rather than re-using existing atomic CSS rules. It exists only as an escape hatch.
